update README: (#1438)
square/sqldelight: README.md

apply plugin: 'com.squareup.sqldelight'
```

> On Android, the plugin will create a default database called `Database` using the project package name. For greater customization, you can declare databases explicitly:

Reviewer: ```suggestion
For Android projects, the plugin will create a default database called `Database` using the project package name. For greater customization, you can declare databases explicitly using the Gradle DSL.
```

> #### build.gradle

Reviewer: ```suggestion
`build.gradle`:
```

```groovy
sqldelight {

>     MyDatabase {

Reviewer: Please use two-space indent for this code block

        /**
         * package name used for the generated MyDatabase.kt
         */
        packageName = "com.example.db"

        /**
         * An array of folders where the plugin will read your '.sq' and '.sqm' files.
         * The folders are relative to the existing source set so if you specify ["db"],
         * the plugin will look into 'src/main/db'
         * Defaults to ["sqldelight"] (src/main/sqldelight)
         */
        sourceFolders = ["db"]

        /**
         * The directory where to store '.db' schema files relative to the root of the project.
         * These files are used to verify that migrations yield a database with the latest schema.
         *
         * Defaults to null so the verification tasks will not be created.
         */
        schemaOutputDirectory = file("src/main/sqldelight/databases")

        /**
         * Optionally specify schema dependencies on other gradle projects

>          */

Reviewer: It's weird that these use KDoc-style and the `linkSqlite` property uses end-of-line comment-style. Please normalize to one or the other.

        dependency project(':OtherProject')
    }

>     // For native targets, chose wether sqlite should be automatically linked.

Reviewer: ```suggestion
    // For native targets, whether sqlite should be automatically linked.
```

    // Defaults to true.
    linkSqlite = false
}
```

> #### build.gradle.kts

Reviewer: Move this below the next sentence and change to

```
`build.gradle.ktx`:
```
similar to what I suggested above. These shouldn't be headers.

> If you're using kotlin for your gradle files:

Reviewer: ```suggestion
If you're using Kotlin for your Gradle files:
```

```kotlin
sqldelight {

>     database("MyDatabase") {

Reviewer: Please use two-space indents for this code block

        packageName = "com.example.db"
        sourceFolders = listOf("db")
        schemaOutputDirectory = file("build/dbs")
        dependency(project(":OtherProject"))
    }
    linkSqlite = false
}
```

Snapshots

> =======

Reviewer: ```suggestion
---------
```
